### PR TITLE
Laboratory work #4, Zhilina Polina - 21FPL2

### PR DESCRIPTION
**Pull request by polyamba**

### CODE REVIEW

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to review:(

File: lab_2_keywords_cooccurrence/target_score.txt [hunk 1]
```diff
@@ -1 +1 @@
-10
\ No newline at end of file
+10
```
Comment (artyomtugaryov) [inline lab_2_keywords_cooccurrence/target_score.txt:1]: revert

File: lab_4_summarization_textrank/main.py [hunk 2]
```diff
@@ -120,23 +163,43 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes
         """
-        pass
+        super().__init__()
+        self.last_id = 0
```
Comment (artyomtugaryov) [inline lab_4_summarization_textrank/main.py:167]: INCUPSULATION

File: lab_4_summarization_textrank/main.py [hunk 3]
```diff
@@ -120,23 +163,43 @@ def __init__(self) -> None:
...
         :param tokens: a sequence of string tokens
         :return:
         """
-        pass
+        if not isinstance(tokens, tuple):
+            raise ValueError
+        for token in tokens:
+            if not isinstance(token, str):
+                raise ValueError
+        # for token, idx in zip(tokens, range(1000, 1000 + len(tokens))):
```
Comment (artyomtugaryov) [inline lab_4_summarization_textrank/main.py:180]: remove

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_4_summarization_textrank/main.py (hunk 3 above)
Comment (polyamba) [reply lab_4_summarization_textrank/main.py]: done

**Pull request closed without merging**